docs/tutorials/network-and-buffering-config.md

```diff
@@ -102,8 +102,8 @@ impose several restrictions on the content that a webpage has access to.
 
 One restriction is [CORS][] (Cross-Origin Resource Sharing).  This requires
 network requests to be made to the same origin, or for the server to explicitly
-give access.  An "origin" refers to the domain name (e.g. `api.example.com`),
-the scheme (e.g. `https:`), and the port (e.g. 80).  If you host your assets on
+give access.  An "origin" refers to the domain name (e.g., `api.example.com`),
```

Comment:
I hadn't seen this particular styling often before, so I looked it up. According to [some grammar sources](https://jakubmarian.com/comma-after-i-e-and-e-g/) this is apparently the way that American style guides recommend, and British style guides recommend no comma. Well, we've been going with American spelling and grammar styles in other places, so I guess we should use this.
Learn something new every day, huh.